modules/sekindoUMBidAdapter.js

```diff
@@ -1,91 +1,113 @@
-      if (bidObj) {
-        var bidCode = bidObj.bidder;
-        var placementCode = bidObj.placementCode;
 
-        if (response.cpm !== undefined && response.cpm > 0) {
-          bid = bidfactory.createBid(CONSTANTS.STATUS.GOOD);
-          bid.callback_uid = callbackId;
-          bid.bidderCode = bidCode;
-          bid.creative_id = response.adId;
-          bid.cpm = parseFloat(response.cpm);
-          bid.ad = response.ad;
-          bid.width = response.width;
-          bid.height = response.height;
+    return validBidRequests.map(bidRequest => {
+      var subId = utils.getBidIdParameter('subId', bidRequest.params);
+      var spaceId = utils.getBidIdParameter('spaceId', bidRequest.params);
+      var bidfloor = utils.getBidIdParameter('bidfloor', bidRequest.params);
+      var protocol = (document.location.protocol === 'https:' ? 's' : '');
+      var queryString = '';
 
-          bidmanager.addBidResponse(placementCode, bid);
-        } else {
-          bid = bidfactory.createBid(CONSTANTS.STATUS.NO_BID);
-          bid.callback_uid = callbackId;
-          bid.bidderCode = bidCode;
-          bidmanager.addBidResponse(placementCode, bid);
+      queryString = utils.tryAppendQueryString(queryString, 's', spaceId);
+      queryString = utils.tryAppendQueryString(queryString, 'subId', subId);
+      queryString = utils.tryAppendQueryString(queryString, 'pubUrl', pubUrl);
+      queryString = utils.tryAppendQueryString(queryString, 'hbTId', bidRequest.transactionId);
+      queryString = utils.tryAppendQueryString(queryString, 'hbBidId', bidRequest.bidId);
+      queryString = utils.tryAppendQueryString(queryString, 'hbver', '4');
+      queryString = utils.tryAppendQueryString(queryString, 'hbcb', '1');/// legasy
+      queryString = utils.tryAppendQueryString(queryString, 'dcpmflr', bidfloor);
+      queryString = utils.tryAppendQueryString(queryString, 'protocol', protocol);
+      if (bidRequest.mediaType === 'video') {
+        queryString = utils.tryAppendQueryString(queryString, 'x', bidRequest.params.playerWidth);
+        queryString = utils.tryAppendQueryString(queryString, 'y', bidRequest.params.playerHeight);
+        if (typeof vid_vastType != 'undefined') {
+          queryString = utils.tryAppendQueryString(queryString, 'vid_vastType', bidRequest.params.vid_vastType);
         }
       }
-    } else {
-      if (bidObj) {
-        utils.logMessage('No prebid response for placement ' + bidObj.placementCode);
+
+      var endpointUrl = 'http' + protocol + '://hb.sekindo.com/live/liveView.php';
+
+      return {
+        method: 'GET',
+        url: endpointUrl,
+        data: queryString,
+      };
+    });
+  },
+  /**
+   * Unpack the response from the server into a list of bids.
+   *
+   * @param {*} serverResponse A successful response from the server.
+   * @return {Bid[]} An array of bids which were nested inside the server.
+   */
+  interpretResponse: function(serverResponse, bidRequest) {
```

Comment:
#1748 changed the first argument of `interpretResponse` to:

```JavaScript
{
  body: responseBody,
  headers: {
    get: function(header) { /* returns a header from the HTTP response */ }
  }
}
```

so adding something like
```JavaScript
serverResponse = serverResponse.body;
```
just below this line, or however you'd prefer to grab the `body`, and updating corresponding tests if needed should get this back to working properly

Reply:
done